dinghy.c

```diff
@@ -274,6 +278,9 @@ on_handle_local_options (GApplication *application,
                                          handler);
     }
 
+    if (s_options.automation_mode)
+        dy_launcher_enable_automation (DY_LAUNCHER (application));
+
     dy_launcher_set_home_uri (DY_LAUNCHER (application), utf8_uri);
```

Comment:
In case of automation mode it's better to ignore any homepage and always load about:blank at startup.

Reply:
I'll make it load `about:blank` when automation is enabled, thanks for the tip.